Core.cpp

```diff
@@ -45,10 +48,18 @@ int main(int, char**)
 
 		if (bboxes.size() && !flag) {
 			//High Beam detected
+
+			detected_image = frame; //save detected frame;
```

Comment:
The `frame` here is cropped etc. Not what we need to save. `detected_image` should ideally be assigned after line 43.